#CreateMap.srv
geometry_msgs/Vector3 gatein
uint32 gateid
---
geometry_msgs/Vector3[40] out_est

